includes/wikia/ExternalUser_Wikia.php

```diff
@@ -330,14 +331,14 @@ public function linkToLocal( $id ) {
 		}
 
 		if( !wfReadOnly() ) { // Change to wgReadOnlyDbMode if we implement that
-			$dbw = wfGetDB( DB_MASTER );
+			$dbw = wfGetDB( DB_MASTER, [], $wgExternalSharedDB );
```

Comment:
In `ExternalUser_Wikia` we actually want to read from a local cluster `wikicities_cX` copy to check whether it actually has user data copy. And if not insert the row there.

Let's leave it as it is right now. It will eventually be simply removed once we refactor all joins to use `wikicities.user` table.

Reply:
Sure, i will bring that back.